src/praisonai/tests/unit/bots/test_presentation_approval_authorization.py

```
@@ -0,0 +1,158 @@
+            await asyncio.sleep(0.01)
+            approval_id = next(iter(handler._pending_approvals))
+            first = await handler.handle_approval_command(
+                approval_id, "allow", actor="admin"
+            )
+            # Duplicate / late callback for the same id
+            second = await handler.handle_approval_command(
+                approval_id, "allow", actor="admin"
+            )
+            return first, second
+
+        result, (first, second) = await asyncio.gather(request(), approver())
```

Ruff (0.15.18)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)